Client/game_sa/CFxSA.cpp

```diff
@@ -11,6 +11,25 @@
 
 #include "StdInc.h"
 
+using StoreShadowToBeRendered_t = int(__cdecl*)(char type, RwTexture* texture, CVector* pos, float x1, float y1, float x2, float y2, __int16 intensity, char r,
+                                                char g, char b, float zDistance, char bDrawOnWater, float scale, void* shadowData, char bDrawOnBuildings);
+auto StoreShadowToBeRendered = (StoreShadowToBeRendered_t)0x707390;
```

Comment:
I think this needs to be done a different way, but I don't know what 0x707390 points to